src/runtime/components/NuxtLinkLocale.ts

```diff
@@ -1,5 +1,7 @@
+import { NuxtLink } from '#components';
```

Comment:
It seems like the linter doesn't check these files, note the semicolons and line length changes.